test/required/requiredexample.proto

```diff
@@ -0,0 +1,14 @@
+package required;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+message RequiredExample {
+    required string theRequiredString = 1;
+    optional string theOptionalString = 2;
+    repeated string theRepeatedStrings = 3;
+}
```

Comment:
I'm unable to do exactly as you ask.

When I add a message
```protobuf
message NidOptNative {
	required double Field1 = 1 [(gogoproto.nullable) = false];
}
```
to requiredexample.proto, I get the following compile error:
```
 make
(protoc --proto_path=../../../../../:../../protobuf/:. --gogo_out=. requiredexample.proto)
2015/05/05 08:18:56 protoc-gen-gogo: error:bad Go source code was generated: 1070:57: expected operand, found ']' (and 2 more errors)
--gogo_out: protoc-gen-gogo: Plugin failed with status code 1.
make: *** [regenerate] Error 1
```

Should I just remove the `[(gogoproto.nullable) = false]` tags (which would make `required.NidOptNative` identical to `required.NinOptNative`), or do you want to figure out why `required` fields clash with setting `nullable` to `false`?